tmail-backend/integration-tests/rspamd/postgres-rspamd-integration-tests/src/test/java/com/linagora/tmail/integration/postgres/PostgresRspamdScannerIntegrationTest.java

```diff
@@ -0,0 +1,41 @@
+package com.linagora.tmail.integration.postgres;
+
+import org.apache.james.JamesServerBuilder;
+import org.apache.james.JamesServerExtension;
+import org.apache.james.PostgresJamesConfiguration;
+import org.apache.james.SearchConfiguration;
+import org.apache.james.backends.postgres.PostgresExtension;
+import org.apache.james.backends.redis.RedisExtension;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.linagora.tmail.blob.blobid.list.BlobStoreConfiguration;
+import com.linagora.tmail.integration.RspamdScannerIntegrationContract;
+import com.linagora.tmail.james.app.PostgresTmailConfiguration;
+import com.linagora.tmail.james.app.PostgresTmailServer;
+import com.linagora.tmail.james.app.RabbitMQExtension;
+import com.linagora.tmail.module.LinagoraTestJMAPServerModule;
+import com.linagora.tmail.rspamd.RspamdExtensionModule;
+
+public class PostgresRspamdScannerIntegrationTest extends RspamdScannerIntegrationContract {
+    @RegisterExtension
+    static JamesServerExtension testExtension = new JamesServerBuilder<PostgresTmailConfiguration>(tmpDir ->
+        PostgresTmailConfiguration.builder()
+            .workingDirectory(tmpDir)
+            .configurationFromClasspath()
+            .searchConfiguration(SearchConfiguration.scanning())
+            .eventBusImpl(PostgresJamesConfiguration.EventBusImpl.RABBITMQ)
+            .blobStore(BlobStoreConfiguration.builder()
+                .postgres()
+                .disableCache()
+                .deduplication()
+                .noCryptoConfig()
+                .disableSingleSave())
+            .build())
+        .extension(PostgresExtension.empty())
+        .extension(new RabbitMQExtension())
+        .extension(new RedisExtension())
```

Comment:
`RspamdExtensionModule` already has a redis container
I guess we don't need duplicate redis containers here

Reply:
I'm not sure they are related in the tests though. The one in RspamdExtensionModule seems dedicated to rspamd, while RedisExtension seems dedicated to rate limiting in tests. 

I could be wrong, but I think we should be a bit cautious here

Reply:
removed. the test still pass